.github/workflows/ci.yaml

```diff
@@ -121,25 +121,30 @@ jobs:
     needs: changes
     if: needs.changes.outputs.gomod == 'true'
     runs-on: ${{ github.repository_owner == 'coder' && 'depot-ubuntu-22.04-8' || 'ubuntu-latest' }}
-    permissions:
-      # Give the default GITHUB_TOKEN write permission to commit and push the changed files back to the repository.
-      contents: write
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 1
+          token: ${{ secrets.CDRCI_GITHUB_TOKEN }}
```

Comment:
This secret is already available in Secrets. I assume that it has push/write access to the repo. We need to add/update a secret if it does not work.

Reply:
This is a PAT for the cdrci account with write perms, so it should be fine writing to PRs on this repo.